### ISSUE TITLE
Confusing warning message from SpaceBeforeFirstArg cop with [] method

### ISSUE THREAD

**Issue by sferik** [rubocop/rubocop#944]
The `SpaceBeforeFirstArg` cop complains about the following code in RuboCop version 0.20.0:

``` ruby
SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter[
  SimpleCov::Formatter::HTMLFormatter,
  Coveralls::SimpleCov::Formatter
]
```

This is the warning message:

```
W: Put space between the method name and the first argument.
```

However, what it really expects is for the spaces to be **removed** after the `[`. Should this be a warning? At the very least, it seems like the wrong message.

If you believe this warning is valid, how would your recommend formatting the above code snippet?
